powa/static/styles/main.scss

```diff
@@ -70,10 +75,18 @@ h6 {
   line-height: 1.3;
   font-size: 0.7rem;
   opacity: 1;
-  background: white;
+  background: var(--v-tooltipbg-base);
   color: inherit;
 }
 
 .v-data-table-header th {
   vertical-align: top;
 }
+
+.theme--light {
+  @include highlight.light;
```

Comment:
Instead of having a copy of highlight theme files in our `static/styles/` isn't it possible to `@include` the corresponding file which already sits in `node_modules` ?

Reply:
@include is only for scss identifier, you can't use it for css files. We can use @import but it's soon to be deprecated in sass.
When we upgrade to vuetify3 and thus sass 1.64, highlight.scss will be unnecessary and we will call meta.load-css("file in node modules") directly.